src/Symfony/Component/Security/Http/Logout/LogoutUrlGenerator.php

```diff
@@ -94,7 +90,7 @@ public function setCurrentFirewall($key, $context = null)
      *
      * @return string The logout URL
      */
-    private function generateLogoutUrl($key, $referenceType)
+    private function generateLogoutUrl(string $key, $referenceType)
```

Comment:
```suggestion
    private function generateLogoutUrl(?string $key, int $referenceType)
```

The firewall key can be null, according to the docblock.